src/components/NavBar/index.js

```diff
@@ -35,27 +22,14 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const NavBar = (props) => {
-  const {
-    home = [],
-    common: { initNavbar = false, navbarError = null },
-  } = props;
+  const { home = [] } = props;
   const classes = useStyles();
-  if (initNavbar) {
-    props.fetchDataInitNavbar();
-  }
-  if (home.length === 0 && navbarError === null) {
-    return (
-      <div align="center">
-        <CircularProgress color="inherit" size={20} />
-      </div>
-    );
-  } else if (navbarError) {
-    return (
-      <div align="center">
-        <p>{navbarError}</p>
-      </div>
-    );
-  }
+
+  useEffect(() => {
+    props.fetchFirebaseData();
```

Comment:
Why do you need to fetch firebase data here?

Reply:
You're right, I don't need to use fetchFirebaseData here. Updated in last commit